configs/GEPRC_TAKER_F405AIO/config.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #pragma once
+
+ #define FC_TARGET_MCU     STM32F405
+ 
+ #define BOARD_NAME        GEPRC_TAKER_F405AIO
+ #define MANUFACTURER_ID   GEPR
+ 
+ #define USE_ACC
+ #define USE_ACC_SPI_ICM42688P
+ #define USE_BARO
+ #define USE_BARO_DPS310
+ #define USE_GYRO
+ #define USE_GYRO_SPI_ICM42688P
+ #define USE_FLASH
+ #define USE_FLASH_W25Q128FV
+ #define USE_MAX7456
+ 
+ #define BEEPER_PIN           PC13
+ #define MOTOR1_PIN           PA8
+ #define MOTOR2_PIN           PA9
+ #define MOTOR3_PIN           PA10
+ #define MOTOR4_PIN           PC8

```

Comment:
Using two GPIO ports is not optimal. See https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines#32-resource-selection-considerations